RFCs/001-indexer.md

```diff
@@ -0,0 +1,126 @@
+# RFC-001: Polka-index
```

Comment:
(from the meeting) -> how can we be sure that a separate p2p network doesn't advertise thrash data?

Reply:
Furthermore — can we simply scrape the chain and make the indexer just a fast way of accessing chain data?